packit.spec

```
@@ -36,6 +36,8 @@ Requires:       rpmdevtools
 # Copying files between repositories
 Requires:       rsync
 
+Requires:       osh-cli
```

Ideally, we want to make the dependency optional. (At least before it's not available in Fedora/EPEL.

```suggestion
Recommends:       osh-cli
```

But we need to check if the `osh` is installed before running the script and provide a meaningful message to the user.

---

I hope this is the correct way to specify this in specfile.